app/lib/pdf_generator.dart

```diff
@@ -21,58 +26,28 @@ class PDFGenerator {
             children: [
               pdf.Text(
                 "Pesticide Record Form",
-                style: pdf.TextStyle(
-                  fontSize: 24,
-                  fontWeight: pdf.FontWeight.bold,
-                ),
-              ),
-              pdf.SizedBox(height: 20),
-
-              pdf.Text(
-                "1. Applicator Name and License Number: ${applicator?.name} - ${applicator?.licenseNumber ?? 'N/A'}",
-              ),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text("2. Name and Address of Owner/Advisor: N/A"),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text(
-                "3. Owner Contacted: ${recordData.customerInformedOfRei ? 'Yes' : 'No'}",
-              ),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text("4. Crop Treated: N/A"),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text(
-                "5. Name and Coordinates of Field: ${recordData.fieldName}",
+                style: pdf.TextStyle(fontSize: 26, fontWeight: pdf.FontWeight.bold),
               ),
               pdf.SizedBox(height: 10),
 
-              pdf.Text("6. Pesticide Name and Registration Number/Rate: N/A"),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text("7. Total Treated Area: N/A"),
-              pdf.SizedBox(height: 10),
-
-              pdf.Text("8. GPA: N/A"),
-              pdf.SizedBox(height: 10),
+              _buildRowWithBottomBox("Applicator Name and License Number", "${applicator?.name} - ${applicator?.licenseNumber}"),

```

Comment:
```suggestion
              _buildInlineRow("Applicator", "${applicator?.name} - ${applicator?.licenseNumber}"),

```

This can probably be an inline box, because the applicator info isn't too long.